docs-source/spring/content/on-premises/_index.md

```diff
@@ -31,23 +31,21 @@ Download the latest release of [Oracle Backend for Spring Boot and Microservices
 
 ## Setup
 
-A custom installation, whether production or non-production, consists of defining the infrastructure followed by running the Configuration
-Management Playbook to build images and deploy the Microservices.
+A custom installation consists of defining the infrastructure followed by running the Configuration Management Playbook to build images and deploy the Microservices.
 
-For a production installation, you need to have a Kubernetes cluster and the `kubectl` command-line interface must be configured to
-communicate with your cluster.
+For a custom installation, you need to have a Kubernetes cluster and the `kubectl` command-line interface must be configured to communicate with your cluster.
 
-A Helper Playbook has been provided for non-production installations to assist in defining the infrastructure.  Review the
-appropriate documentation for examples of installing and defining the non-production installation. For example:
+A Helper Playbook has been provided for desktop installations to assist in defining a known infrastructure.  Review the
```

Comment:
i still think this needs to be made super clear - i read "desktop" and think, yeah i'm on my desktop, but that's not what you are trying to say - you are trying to say if you happen to be setting it up with minikube,podman, etc., exactly as expected.  we need to make it super clear so people knwo when to use the desktop_apply and when to skip it